src/platform/plugins/shared/home/public/application/components/tutorial/status_check_states.ts

```diff
@@ -7,7 +7,7 @@
  * License v3.0 only", or the "Server Side Public License, v 1".
  */
 
-export const HAS_DATA = 'has_data';
+export const HAS_DATA = 'HAS_DATA';
```

Comment:
I double checked that this change seems safe, because this isn't coming from the server